compiler/rustc_hir_analysis/src/impl_wf_check/min_specialization.rs

```diff
@@ -258,23 +258,23 @@ fn unconstrained_parent_impl_args<'tcx>(
     // unconstrained parameters.
     for (clause, _) in impl_generic_predicates.predicates.iter() {
         if let ty::ClauseKind::Projection(proj) = clause.kind().skip_binder() {
-            let projection_ty = proj.projection_ty;
-            let projected_ty = proj.term;
+            let projection_term = proj.projection_term;
+            let projected_term = proj.term;
```

Comment:
if you want: just remove these variables and replace the uses with their initialisers